bindings/python/nanobind.BUILD

```diff
@@ -33,6 +44,7 @@ cc_library(
         "include",
     ],
     linkopts = select({
+        ":winplusmsvc": ["/LTGC"],  # allows other compilers on Windows than just MSVC.
```

Comment:
so clang on windows takes /LTGC as a parameter?

Reply:
Ah sorry, I guess that came from the comment. The `:winplusmsvc` condition only applies for Windows + MSVC, so for Windows + clang, it falls through to `//conditions:default`, which is no options.

I can update the comment if it's too misleading.